diff -ur silo-4.10.2-bsd.orig/src/hdf5_drv/silo_hdf5.c silo-4.10.2-bsd/src/hdf5_drv/silo_hdf5.c
--- silo-4.10.2-bsd.orig/src/hdf5_drv/silo_hdf5.c	2014-10-13 19:22:32.000000000 -0500
+++ silo-4.10.2-bsd/src/hdf5_drv/silo_hdf5.c	2021-09-07 20:54:38.889145320 -0500
@@ -4755,7 +4755,7 @@
         case DB_FILE_OPTS_H5_DEFAULT_MPIP:
         {
 #ifdef H5_HAVE_PARALLEL
-            h5status |= H5Pset_fapl_mpiposix(retval, MPI_COMM_SELF, TRUE);
+            h5status |= H5Pset_fapl_mpio(retval, MPI_COMM_SELF, MPI_INFO_NULL);
 #else
             H5Pclose(retval);
             return db_perror("HDF5 MPI VFD", E_NOTENABLEDINBUILD, me);
@@ -5076,7 +5076,7 @@
                     }
                     else
                     {
-                        h5status |= H5Pset_fapl_mpiposix(retval, mpi_comm, use_gpfs_hints);
+                        h5status |= H5Pset_fapl_mpio(retval, mpi_comm, MPI_INFO_NULL);
                     }
 #else 
                     H5Pclose(retval);
