diff -ur silo-4.11.orig/src/hdf5_drv/silo_hdf5.c silo-4.11/src/hdf5_drv/silo_hdf5.c
--- silo-4.11.orig/src/hdf5_drv/silo_hdf5.c	2021-09-09 14:35:00.000000000 -0500
+++ silo-4.11/src/hdf5_drv/silo_hdf5.c	2023-02-28 21:26:02.000000000 -0600
@@ -58,8 +58,6 @@
    version 1.8 and thereafter. When, and if, the HDF5 code in this file
    is explicitly upgraded to the 1.8 API, this symbol should be removed. */
 #define H5_USE_16_API
-#warning DO WE NEED THIS HEADER FILE
-#include <H5pubconf.h>
 #include <hdf5.h>
 
 #include <errno.h>
