diff -ur ../../../../../repos/ExternalLibraries-Silo/dist/silo-4.11/src/hdf5_drv/H5Zzfp.c silo-4.11/src/hdf5_drv/H5Zzfp.c
--- ../../../../../repos/ExternalLibraries-Silo/dist/silo-4.11/src/hdf5_drv/H5Zzfp.c	2020-12-08 22:59:30.000000000 -0600
+++ silo-4.11/src/hdf5_drv/H5Zzfp.c	2023-01-13 11:34:13.833542586 -0600
@@ -1,6 +1,8 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef HAVE_ZFP
+
 /*
 This code was based heavily on one of the HDF5 library's internal
 filter, H5Zszip.c. The intention in so doing wasn't so much to 
@@ -642,3 +644,4 @@
 
 #undef Z
 #undef B
+#endif /* HAVE_ZFP */
diff -ur ../../../../../repos/ExternalLibraries-Silo/dist/silo-4.11/src/hdf5_drv/silo_hdf5.c silo-4.11/src/hdf5_drv/silo_hdf5.c
--- ../../../../../repos/ExternalLibraries-Silo/dist/silo-4.11/src/hdf5_drv/silo_hdf5.c	2021-09-09 14:35:00.000000000 -0500
+++ silo-4.11/src/hdf5_drv/silo_hdf5.c	2023-01-13 11:30:28.148514093 -0600
@@ -3341,8 +3341,10 @@
             have_fpzip = TRUE;
         if (DB_HDF5_HZIP_ID==filtn)
             have_hzip = TRUE;
+#ifdef HAVE_ZFP /* { */
         if (H5Z_FILTER_ZFP==filtn)
             have_zfp = TRUE;
+#endif
     }
 #warning WHAT ABOUT NULL RETURN FROM DBGETCOMPRESSION
 /* Handle some global compression parameters */
